cellprofiler/gui/help.py

```diff
@@ -2027,287 +2027,323 @@
+            <li>Make a new rule <code>[Metadata][Does][Have Stain matching][phase]</code> and name it
+            <i>OrigPhase</i>.</li>
+            <li>In the "Image set matching method" setting, select "Metadata".</li>
+            <li>Select "Well" for the <i>OrigFluor</i>, <i>OrigFluo2</i>, and <i>OrigPhase</i> channels.</li>
+            <li>Click the <img src="memory:{MODULE_ADD_BUTTON}">&nbsp;button to the right to add another row, and
+            select "Timepoint" for each channel.</li>
+            <li>Click the "Update" button below the divider to view the resulting table and confirm that the proper
+            files are listed and matched across the channels. The corresponding well and frame for each channel should
+            now be matched to each other.</li>
+        </ul>
+    </li>
+    <li>In the <b>Groups</b> module, enable image grouping for these images in order to select the metadata that
+    defines a distinct movie of data.<br>
+        For the example above, do the following:
+        <ul>
+            <li>Select "Well" as the metadata category.</li>
+            <li>The tables below this setting will update themselves, and you should be able to visually confirm that
+            each well is defined as a group, each with 287 frames' worth of images.</li>
+        </ul>Without this step, CellProfiler would not know where one movie ends and the next one begins, and would
+        process the images in all movies together as if they were a single movie. This would result in, for example,
+        the TrackObjects module attempting to track cells from the end of one movie to the start of the next movie.
+    </li>
 </ul>
-</li>
-<li>In the <b>Groups</b> module, enable image grouping for these images in order to select the metadata
-that defines a distinct movie of data.</br>
-For the example above, do the following:
-<ul>
-<li>Select "Well" as the metadata category.</li>
-<li>The tables below this setting will update themselves, and you should be able to visually confirm that
-each well is defined as a group, each with 287 frames' worth of images.</li>
-</ul>
-Without this step, CellProfiler would not know where one movie ends and the next one begins, and would
-process the images in all movies together as if they were constituents of only one movie.
-</li>
-</ul>
-</p>
-
+If your images represent a volume, you can follow the above example to process your volume. It is important to note,
+however, that CellProfiler will analyze each Z-slice image individually and sequentially. If you would like to analyze 
+each Z-stack as a whole
+volume, we suggest building a .TIF stack from the individual image files using another software application, like FIJI
+and then using the instructions below for single-file sequences.
+Additionally, whole volume processing is only supported for single-channel volumes. If your images are multi-channel
+you will need to separate the channels before combining images into a single-image stack.
```

Comment:
Suggested rewrite of that paragraph:

"If your image series represents a 3D image, while you _can_ essentially follow the instructions for the grouping of time series, it will mean that your individual z slices will be processed separately rather than as one 3D whole.  If you want to analyze the series as a whole, we suggest building a .TIF stack from the individual image files using another software application, like FIJI and then using the instructions below for single-file sequences.  Please note that 3D images should be separated into their individual channels, ie one 3D .TIF stack for the DNA channel, one 3D .TIF stack for the actin channel, etc."

We're going to want to either write or point them to such a tutorial for making such a series as well.